src/modules/dashboard/components/ColonyHomeActions/ColonyActionsDialog.css

```diff
@@ -0,0 +1,20 @@
+.content {
+  padding-bottom: var(--pad-huge);
```

Comment:
This needs a bottom padding of `60px` as per the design file in figma

![Screenshot from 2020-11-11 10-59-49](https://user-images.githubusercontent.com/1193222/98791057-2c386400-240d-11eb-8f2e-e231dc4532c5.png)

Since this will be a value that we don't normally use in the app, you can set it locally in this file via a `@value` declaration.

Eg:
```css
@value customPadding: 60px;
```
Then use it like this:
```css
.content {
  padding-bottom: customPadding;
}
```